homeassistant/components/tasmota/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Tasmota",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/tasmota",
-  "requirements": ["hatasmota==0.0.1"],
+  "requirements": ["hatasmota==0.0.4"],
```

Comment:
How we did already hit 0.0.4 Erik? :D 
Speaking of the version, should Tasmota follow the integration version? Now is hardcoded as `1`.